### PR TITLE
Basic front end build and tests

### PR DESCRIPTION
**Pull request by jwhunt19**

### CODE REVIEW

**Comment by jwhunt19** [comment]
[Set up Front End and Testing](https://trello.com/c/QcuJ4eB6/33-set-up-front-end-and-testing)

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          product.images
+          && (
+            <Photos
+              changeSelectedImg={this.changeSelectedImg}
+              images={product.images}
+              selectedImg={selectedImg}
+            />
+          )
+        }
+        <Sidebar product={product} seller={seller} />
```
Comment (jclasley) [inline client/src/components/App.jsx:62]: Why not make the conditional here? Instead of inside `Sidebar`

File: client/src/components/Sidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  product: PropTypes.shape({
+    id: PropTypes.number,
+    name: PropTypes.string,
+    image: PropTypes.string,
+    description: PropTypes.string,
+    price: PropTypes.string,
+    stock: PropTypes.number,
+    sizes: PropTypes.bool,
+    rating: PropTypes.number,
+  }).isRequired,
```
Comment (jclasley) [inline client/src/components/Sidebar.jsx:109]: Oof

**Review by jclasley — APPROVED** [review]
You will have to refactor your `classNames` to use CSS modules somewhere down the line. I know that you aren't doing CSS yet so it's nbd, just a heads up.

**Pull request merged**